lfs/upload_test.go

```diff
@@ -923,3 +923,11 @@ func TestUploadVerifyError(t *testing.T) {
 		t.Errorf("verify not called")
 	}
 }
+
+func uploadMediaPath(t *testing.T, oid string) string {
```

Comment:
Hmm. This seems like a solid approach to take to determine when to skip tests, since all of the skipped tests require an OID path, but I'm not sure the side-effect of skipping a test is immediately obvious from this method's name.

Another idea for this would be to leave the OID path stuff as it was before this PR, and introduce a different method:

```go
func AssertInRepository(t *testing.T) {
        if objects == nil {
                t.Skip("Not in a git repository")
        }
}
```

I think I prefer this other method more since it has explicit behavior, and has no return type, so no unintended side-effects.

Not a big deal either way, but just my $.02. 😄